app/assets/stylesheets/components/_form.pcss

```
@@ -0,0 +1,47 @@
+.form {
+  &__group,
+  &__actions {
```

Avoid plural class names:

```suggestion
  &__action {
```